pkg/proto/configuration/cloud/aws/aws.proto

```diff
@@ -30,4 +30,13 @@ message SessionConfiguration {
   // will search the default credential provider chain (e.g.,
   // environment variables, EC2 instance IAM roles).
   StaticCredentials static_credentials = 5;
+
+  // Log levels are documented at
+  // https://docs.aws.amazon.com/sdk-for-go/api/aws/#LogLevelType
+  bool log_debug = 6;
```

Comment:
Would it make sense to use a `repeated enum` instead?
```go
enum LogLevelType {
  DEBUG = 1;
  DEBUG_WITH_WHATEVER = 2;
  ...
}
repeated LogLevelType log_level = 6;
```
That way it's easier to parameterize/forward/... log levels without needing to resort to merging objects.